lib/sinon/collection.js

```diff
@@ -46,7 +46,13 @@ var collection = {
     },
 
     resetHistory: function resetHistory() {
-        each(this, "resetHistory");
+        getFakes(this).forEach(function (fake) {
+            var method = fake.resetHistory || fake.reset;
+
+            if (method) {
```

Comment:
Which is a bit synthetic right now .... I could also just call `fake.reset`, since that is what stub.resetHistory does ... but that just seems *wrong*